src/data/staking-products.json

```diff
@@ -636,6 +636,35 @@
         "eventAction": "Clicked",
         "eventName": "Clicked Sensei Node go to link"
       }
+    },
+    {
+      "name": "Figment",
+      "imageName": "Figment",
+      "hue": 52,
+      "launchDate": "2021-12-31",
+      "url": "https://figment.io/",
+      "audits": [],
+      "minEth": 32,
+      "monthlyFee": 30,
+      "monthlyFeeUnit": "% execution rewards",
+      "isFoss": false,
+      "hasBugBounty": false,
```

Comment:
@ivanszeftel As noted, this is from it being private
<img width="598" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/153380b2-d552-4689-a26d-57a453f3c28e">